desk/app/genuine.hoon

```diff
@@ -0,0 +1,87 @@
+/+  default-agent, verb, dbug, server
+::
+|%
+++  give-payload
+  |=  [id=@ta =simple-payload:http]
+  (give-simple-payload:app:server id simple-payload)
+++  make-secret
+  |=  eny=@uvJ
+  ^-  cord
+  (scot %uw eny)
++$  card  card:agent:gall
++$  versioned-state
+  $%  state-0
+  ==
++$  state-0
+  $:  %0
+      secret=cord
+  ==
+--
+=|  state-0
+=*  state  -
+%-  agent:dbug
+%+  verb  |
+^-  agent:gall
+|_  =bowl:gall
++*  this  .
+    def   ~(. (default-agent this %.n) bowl)
+::
+++  on-init
+  :_  this(secret (make-secret eny.bowl))
+  ~[[%pass /eyre/connect %arvo %e %connect [~ /genuine] dap.bowl]]
```

Comment:
We might consider binding on `/apps/landscape/genuine` if we're not distributing this as a stand-alone application, just so we aren't squatting multiple top-level path element.